backend/src/test/java/com/woowacourse/gongseek/acceptance/AcceptanceTest.java

```diff
@@ -0,0 +1,18 @@
+package com.woowacourse.gongseek.acceptance;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public abstract class AcceptanceTest {
+    @LocalServerPort
+    private int port;
```

Comment:
클래스 선언하고 한 줄은 공백으로 두는게 어떠신가요?
```suggestion
public abstract class AcceptanceTest {

    @LocalServerPort
    private int port;
```